packages/web/server/utils/embeddings.ts

```diff
@@ -152,6 +152,6 @@ async function generateEmbedding(text: string): Promise<number[]> {
   }
 }
 
-function getLabels(issue: Issue | RestIssue) {
+function getLabels(issue: Pick<Issue | RestIssue, 'labels' | 'title' | 'body'>) {
```

Comment:
NIT: And you get to re-use it down here:
```suggestion
function getLabels(issue: Pick<Issue | RestIssue, IssueSegments >) {
```